hw_twitterapi/twitterapiapp/templates/twitterapiapp/friends.html

```diff
@@ -0,0 +1,34 @@
+<title>Hilal's Twitter app</title>
```

Comment:
We have a django template having a base file to keep the theme the same all over the project. Please make necessary changes. Refer to other user's files.